openlibrary/macros/AvailabilityButton.html

```diff
@@ -66,7 +66,6 @@
   <form>
     <input type="button" class="cta-btn--missing cta-btn" disabled value="No ebook available">
   </form>
-  $if page.ia:
+  $if availability_status != 'error' and page.ia:
```

Comment:
I think it would be more robust to test for success here rather than not error.

Reply:
Actually going to revert this change; this was here because I think books like this https://archive.org/details/jungle00sinc_982 report as "not found" from the availability API https://archive.org/services/availability/?identifier=jungle00sinc_982 (for some reason). I think those are more common than completely incorrect IA ids.